client/rpc/debug/src/lib.rs

```diff
@@ -236,11 +236,25 @@ where
 		if let Some(block) = reference_block {
 			let transactions = block.transactions;
 			if let Some(transaction) = transactions.get(index) {
-				return client
-					.runtime_api()
-					.trace_transaction(&parent_block_id, ext, &transaction, trace_type)
-					.map_err(|e| internal_err(format!("Runtime api access error: {:?}", e)))?
-					.map_err(|e| internal_err(format!("DispatchError: {:?}", e)));
+				let f = || {
+					return client
```

Comment:
```suggestion
					client
```

Explicit return is not necessary right? (will also require removing the semicolon.